frontend/test/pytest/test_jax_dynamic_api.py

```diff
@@ -980,5 +1082,29 @@ def case():
     assert_array_and_dtype_equal(f(False, 3), jnp.zeros([4, 3]))
 
 
+def test_qjit_cond_capture():
+    """Test that catalyst tensor primitive is compatible with quantum conditional"""
+
+    @qjit
+    def f(flag, sz):
+        a = jnp.ones([sz, 3], dtype=float)
+
+        @cond(flag)
+        def case():
+            b = jnp.ones([sz, 3], dtype=float)
+            return a + b
+
+        @case.otherwise
+        def case():
+            b = jnp.ones([sz, 3], dtype=float)
```

Comment:
```suggestion
            b = jnp.zeros([sz, 3], dtype=float)
```

Just to make sure that there is a difference.